thoth/storages/inspection_schema.py

```diff
@@ -0,0 +1,84 @@
+#!/usr/bin/env python3
+# thoth-storages
+# Copyright(C) 2019 Fridolin Pokorny, Francesco Murdaca
+#
+# This program is free software: you can redistribute it and / or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program. If not, see <http://www.gnu.org/licenses/>.
+
+"""Schema definition for inspection results."""
+
+from voluptuous import Required
+from voluptuous import Schema
+
+class Datetime(object):
```

Comment:
Can you use the one defined in `result_schema.py`? Copying code is bad practice as we will need to maintain the same thing on multiple places.

Reply:
Corrected thanks!